flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -708,6 +708,11 @@ public class TaskManagerOptions {
                                                     "The %s mode tries to spread out the slots evenly across all available %s.",
                                                     code(TaskManagerLoadBalanceMode.SLOTS.name()),
                                                     code("TaskManagers")),
+                                            text(
+                                                    "The %s mode tries to allocate slots on minimum number of available resources.",
+                                                    code(
+                                                            TaskManagerLoadBalanceMode.RESOURCES
```

Comment:
```suggestion
                                                            TaskManagerLoadBalanceMode.MIN_RESOURCES
```
I like the naming you found in the test implementation. That seems to be more descriptive than just "resources" in the context of load balancing. Can we change that here as well? WDYT?